address/book.go

```diff
@@ -70,6 +70,20 @@ type QueryParams struct {
 	UnmanagedOnly bool
 }
 
+// AssetSyncer is an interface that allows the address.Book to look up asset
+// genesis and group information from both the local asset store and assets
+// known to universe servers in our federation.
+type AssetSyncer interface {
+	// FetchAssetInfo queries the universes in our federation for genesis
+	// and asset group information about the given asset ID.
+	FetchAssetInfo(ctx context.Context, assetID *asset.ID) error
```

Comment:
IIUC, we just want to attempt to sync it we get an error that we don't know of the asset ID on disk. So we don't need to check to see if we know of it, if we just try and then detect the error when we don't.

Reply:
Resolved later on? That's what we do with `queryAssetInfo`, intent was to keep `QueryAssetGroup` calls separate from those backend by the universe interfaces, and leave room to turn off this JIT bootstrapping. Also I think sync would fail if we minted the asset and never pushed issuance proofs to any universe in our federation.